saleor/graphql/product/mutations/products.py

```diff
@@ -1656,6 +1540,7 @@ def perform_mutation(cls, _root, info, product_id, images_ids):
         product = cls.get_node_or_error(
             info, product_id, field="product_id", only_type=Product
         )
+        product_variant_exist(product)
```

Comment:
Reordering product images seems independent from variants, why do we disallow it here?

Reply:
good catch, thanks. I'll remove this.